UI/UnoMaterialSample/UnoMaterialSample.Skia.Gtk/UnoMaterialSample.Skia.Gtk.csproj

```diff
@@ -14,12 +14,14 @@
     <UpToDateCheckInput Include="..\UnoMaterialSample.Shared\**\*.xaml" />
   </ItemGroup>
   <ItemGroup>
-    <!-- Note that for WebAssembly version 1.1.1 of the console logger required -->
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.1" />
-    <PackageReference Include="Uno.Material" Version="1.0.0-dev.780" />
-    <PackageReference Include="Uno.UI.Skia.Gtk" Version="3.8.6" />
-    <PackageReference Include="Uno.UI.RemoteControl" Version="3.8.6" Condition="'$(Configuration)'=='Debug'" />
+    <PackageReference Include="Uno.Toolkit.UI" Version="1.3.0" />
+    <PackageReference Include="Uno.Toolkit.UI.Material" Version="1.3.0" />
+    <PackageReference Include="Uno.UI" Version="4.1.9" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
+    <PackageReference Include="Uno.UI.Skia.Gtk" Version="4.1.9" />
```

Comment:
Looks like you're missing he Uno.Material PackageReference

Reply:
I added that now. But didn't have any difference. It runs exactly the same way, with our without. Same appearence, no errors.

Reply:
Oh you know what, my bad. I didn't realize this was the Skia.Gtk head, so you only need the Uno.UI.Skia.Gtk package which was already there.